src/test/mir-opt/issue-49232.rs

```diff
@@ -0,0 +1,186 @@
+// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// compile-flags: -Z identify_regions -Z emit-end-regions
```

Comment:
Do we need these two lines? (cc @nikomatsakis @arielb1) I expect the MIR to be shorter and easier to read without them.

Reply:
Done-- made it a bit shorter.